modules/react-components/src/components/tab/resource-tab/resource-tab.tsx

```diff
@@ -123,60 +127,60 @@ export const ResourceTab: FunctionComponent<ResourceTabPropsInterface> & Resourc
         setActiveIndex(activeIndex);
     };
 
-    if (isLoading === false) {
+    if (isLoading) {
         return (
-            <Tab
-                onTabChange={ (e: MouseEvent<HTMLDivElement>, data: TabProps) => {
-                    handleTabChange(e, data.activeIndex);
-                    onTabChange && typeof onTabChange === "function" && onTabChange(e, data);
-                } }
-                className={ classes }
-                menu={ {
-                    attached,
-                    pointing,
-                    secondary
-                } }
-                panes={ panes }
-                activeIndex={ activeIndex }
-                data-componentid={ componentId }
-                data-testid={ testId }
-                { ...rest }
-            />
+            <>
+                <Card.Group style={ { marginBottom: "3rem" } }>
+                    {
+                        [ ...Array(loadingStateOptions?.count ? loadingStateOptions?.count : 0) ].map(() => {
```

Comment:
`loadingStateOptions?.count ? loadingStateOptions?.count : 0`
We can replace this as follows,
`loadingStateOptions?.count ?? 0`

Reply:
FIxed with - https://github.com/wso2/identity-apps/pull/3629/commits/a7ed6224d16b34839c9ace15068a1d13d614f898